### PR TITLE
Additional provider validation

### PR DESCRIPTION
**Pull request by jdziurzynski**
KNTK-390

Added validation to few fields on provider schema.

### CODE REVIEW

**Review by mmac-m3a — APPROVED** [review]
LGTM

**Comment by mmac-m3a** [comment]
I'll wait for @danielfurman to review the PR before merging it.

File: internal/provider/cloudexport_schema.go
Comment (danielfurman) [inline internal/provider/cloudexport_schema.go:90]: Could you explain what is the purpose of that? And why we have 4 values here?
I understand that this is a 1-element list of AWS object, so I don't know how ExactlyOneOf applies to this.

Comment (danielfurman) [reply internal/provider/cloudexport_schema.go:90]: As discussed in other channel, this is for validation that there is only one object with provider properties. We need to use separate awsKey constant to make it readable.

**Review by danielfurman — CHANGES_REQUESTED** [review]

File: internal/provider/cloudexport_schema.go
Comment (danielfurman) [reply internal/provider/cloudexport_schema.go]: Done

**Review by danielfurman — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

**Pull request merged**
